src/templates/drawer.jsx

```diff
@@ -416,7 +423,7 @@ class Drawer extends React.Component {
                             autoHide
                         >
                             <div className="drawer-container-inner">
-                                {children}
+                                    {children}
```

Comment:
Looks like we added extra whitespace here?  Any reason?  Could we revert that?

Reply:
Thanks @groberts314 , for catching it. Fixed.